tests/RepositoryForms/FieldType/Mapper/FormTypeBasedFieldValueFormMapperTest.php

```diff
@@ -44,6 +54,11 @@ public function testMapFieldValueFormWithLanguageCode()
             ->with('fieldDefinition')
             ->willReturn($fieldDefinition);
 
+        $this->config->expects($this->once())
```

Comment:
Is it meant to be called exactly **once**? It is usually not true for getters. It will make test fail after minor and not destructive change in the code i.e. by calling `getOption` twice.

Reply:
Well, it is just moved from here, https://github.com/ezsystems/repository-forms/pull/263/files/bea16be48cc553188c00d1455fcab83280ff02ea..2c631b8525d4243f2ba09bee5e170f18af25a9cb#diff-7cfca92e908e07bb6ecbe1da7e5cb107L30
so not sure about it